uniffi/src/templates/macros.rs

```diff
@@ -24,9 +24,68 @@
 // Arglist as used in the _UniFFILib function declations.
 // Note unfiltered name but type_c filters.
 -#}
-{%- macro arg_list_rs_decl(args) %}
-    {%- for arg in args %}
-        {{- arg.name() }}: {{ arg.type_()|type_c -}}
-        {%- if !loop.last %}, {% endif %}
+{%- macro arg_list_rs_decl(func) %}
+    {%- for arg in func.arguments() %}
+        {{- arg.name() }}: {{ arg.type_()|type_c -}}{% if loop.last %}{% else %},{% endif %}
     {%- endfor %}
+    {% if func.has_out_err() %}
+    {% if func.arguments().len() > 0 %},{% endif %} err: &mut ffi_support::ExternError,
```

Comment:
Holy cow, so many commas to get right! 😅

Reply:
Ya for sure can make it cleaner 👀